src/AntiCSRF.php

```diff
@@ -52,9 +52,25 @@ class AntiCSRF
     const SESSION_INDEX = 'CSRF';
     const HASH_ALGO = 'sha256';
 
-    public static $recycle_after = 65535;
-    public static $hmac_ip = true;
-    public static $expire_old = false;
+    public $recycle_after = 65535;
+    public $hmac_ip = true;
+    public $expire_old = false;
```

Comment:
can't we make this as protected / private . I don't think we really need these variables to be called outside. Let me know what you guys think.